tests/manual-tests/framework/preferences/assortedContent/index.html

```diff
@@ -291,6 +291,119 @@ <h4>Mid-term Exam</h4>
                             <textarea>Please enter a simple JavaScript function to find the smallest value of an array.</textarea>
                             <button>Run Script</button>
                         </fieldset>
+                        <fieldset class="disabled-inputs">
+                            <legend>HTML Property</legend>
```

Comment:
I think that we are either missing a layer of `fieldset` here or we need to change this `legend`. At the moment it looks like the whole disabled inputs `fieldset` is called "HTML Property" rather than something like "Disabled Inputs".